src/Compilers/Core/Portable/PEWriter/CompilationOptionNames.cs

```diff
@@ -37,5 +37,6 @@ internal static class CompilationOptionNames
         public const string OptionInfer = "option-infer";
         public const string OptionExplicit = "option-explicit";
         public const string OptionCompareText = "option-compare-text";
+        public const string OutputKind = "output-kind";
```

Comment:
>output [](start = 42, length = 6)

Can't the exact kind be inferred from the headers? It's not ideal when we duplicate information that's already there.

Reply:
I was unable to find a way to do this. If you can see a way I'm happy to flip to using that but there didn't seem to be the necessary info in the headers for all of the output kinds that we support.

Reply:
My bad. I forgot we had put so much detail into that doc. It's out of date for a lot of changes I made. I will send a PR to get this current.